ledger/src/shred.rs

```diff
@@ -393,6 +395,7 @@ impl Shred {
     dispatch!(pub fn set_index(&mut self, index: u32));
     dispatch!(pub fn set_slot(&mut self, slot: Slot));
 
+    #[cfg_attr(feature = "dev-context-only-utils", qualifiers(pub))]
```

Comment:
Should below line be ? If it still has the non-conditional `pub`, I'm not sure if it is actually hidden or not
```rust
fn copy_to_packet(&self, packet: &mut Packet) {
```

Reply:
Good point, fixed

Reply:
Actually, it has to be `pub`, because tests in `ledger/src/sigverify_shreds.rs` are using it.

So it should be public, but hidden behind `#[cfg(any(test, feature = "dev-context-only-utils"))]`.